src/components/TransactionConfirmationModal/index.tsx

```diff
@@ -183,26 +183,26 @@ export function ConfirmationModalContent({
   bottomContent?: () => ReactNode | undefined
 }) {
   return (
-    <Wrapper>
-      <Section>
-        <RowBetween>
-          <Text fontWeight={500} fontSize={16}>
+    <ConfirmationModalWrapper>
+      <AutoColumn gap="0.5rem" padding="1em 0.5rem 0">
+        <Row>
+          <BackArrowIcon style={{ position: 'absolute' }} onClick={onDismiss} data-cy="confirmation-close-icon" />
+          <Text fontWeight={500} fontSize={16} marginX="auto">
```

Comment:
Sounds like `ThemedText.SubHeader`. Also, what is the purpose of `marginX` here? Centering text? Can we/should we do it with `Row` flex property instead?